Tasks/intersection.js

```diff
@@ -1,25 +1,14 @@
 // Find an intersection of two dictionaries
 
-intersection = function intersection(object_1, object_2) {
-  first_keys = Object.keys(object_1);
-  {
-  }
-  for (attribute_name of first_keys) {
-    if (object_1[attribute_name] === object_2[attribute_name]) {
-      object_2[attribute_name] = object_1[attribute_name];
-      {
-      }
-    } else {
-      delete object_1[attribute_name];
-      {
-      }
-    }
-    {
-    }
-  }
-  {
+'use strict';
+
+const intersection = (object1, object2) => {
+  for (const key of Object.keys(object1)) {
+    object1[key] === object2[key]
+      ? (object2[key] = object1[key])
+      : delete object1[key];
```

Comment:
Use ternary only for expressions to assignin/pass a result of calculations somewhere